cypress/e2e/collections-spec.cy.js

```diff
@@ -0,0 +1,37 @@
+describe('should be able to view collections', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=5&sort=hot',
+      {
+       statusCode:201,
+       fixture: 'trending.json'
+      }
+    )
+
+    cy.visit('http://localhost:3000/trending')
+  })
+
+  it('should have a saved collection', () => {
+    cy.get('.banner-container').contains('h1', 'SOUND STASH')
+      .get('.banner-container').contains('a', 'DISCOVER')
+      .get('.recommendedContainer').find('.link').should('have.length', 5).first().find('img').should('have.attr', 'src')
+      .get('.recommendedContainer').find('.link').first().find('.album').contains('p', 'Queens')
+      .get('.recommendedContainer').find('.link').last().find('img').should('have.attr', 'src')
+      .get('.recommendedContainer').find('.link').last().find('.album').contains('p', 'Akira')
```

Comment:
curious as to why you're testing these in the collections spec?

Reply:
Probably a good idea to move it to a trending file... I'll do that as well as add sad paths for 404 & 500 level errors.

Reply:
we gonna fix this up in a follow-on for polishing testing